automation/test-windows.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
```

Comment:
could you please elaborate why do we need this file in the RHEL8 PR?

Reply:
this file was created, because previously we had only one test for windows. With new rhel test i needed to separate those two tests.